source/Octopus.Diagnostics/ILog.cs

```diff
@@ -5,32 +5,46 @@ namespace Octopus.Diagnostics
     public interface ILog
     {
         void Trace(string messageText);
+        void Trace(string messageText, params object[] propertyValues);
```

Comment:
I've added new methods instead of modifying existing ones to maintain backward compatibility